Kentico.Kontent.Delivery.Abstractions/SharedModels/IError.cs

```diff
@@ -0,0 +1,28 @@
+﻿namespace Kentico.Kontent.Delivery.Abstractions
+{
+    /// <summary>
+    /// Represents a faulty JSON response from Kentico Kontent Delivery API.
+    /// </summary>
+    public interface IError
+    {
+        /// <summary>
+        /// Gets error Message.
+        /// </summary>
+        string Message { get; }
+
+        /// <summary>
+        /// Gets request id.
```

Comment:
how about something like "Gets the ID of a request that can be used for troubleshooting"?

Reply:
Thank you for the suggestion, updated.